### DISCUSSION TITLE
감사합니다.

### DISCUSSION THREAD

**Discussion by anno4** [changdoc/naver-vaccine-macro#83] [General]
감사인사글 남기려고 github 가입 어렵게 했네요. 
사이트 시용이 생소하네요. 이렇게 남겨도 되는 것인지..
아무튼 금일 오전 10시경 갑자기 빵빠레 소리가 나서
깜짝 놀랐었습니다.
손쉽게 만들어 주셔서 감사합니다.
건강 유의 하시고 늘 행복하세요!

**Comment by seokwonchoisnu** [comment]
저는 오늘 아침부터 한개의 병원에서 자동예약 돌렸는데 안되고 있는데 이렇게 한개의 병원만 하면 되나요?

**Comment by soosooho** [comment]
운이 좋으시네요 ㅎㅎ 축하드립니다
